client/index.coffee

```diff
@@ -53,7 +53,7 @@ bitcamp = angular.module("bitcampApp", [
     }
 
     $rootScope.ready = false
-    $http.get("/api/bitcamp")
+    $http.get("/")
```

Comment:
cuz github pages doesn't run an API.. this makes the text look nice after pageload and reflow